Sources/Subprocess/IO/Output.swift

```diff
@@ -240,10 +240,9 @@ extension OutputProtocol where Self == FileDescriptorOutput {
 @available(SubprocessSpan, *)
 #endif
 extension OutputProtocol where Self == StringOutput<UTF8> {
-    /// Create a `Subprocess` output that collects output as
-    /// UTF8 String with 128kb limit.
+    /// Create a `Subprocess` output that collects output as UTF8 String
```

Comment:
I think the documentation comment here should say that the buffer is unlimited, so the amount of memory required is proportional to the size of the output.

Reply:
Sounds good